common/src/main/kotlin/com/hartwig/actin/configuration/ReportConfiguration.kt

```diff
@@ -0,0 +1,71 @@
+package com.hartwig.actin.configuration
+
+import com.hartwig.actin.datamodel.molecular.evidence.Country
+
+enum class ReportContentType {
+    NONE,
+    BRIEF,
+    COMPREHENSIVE
+}
+
+enum class MolecularChapterType {
+    DETAILED_WITHOUT_PATHOLOGY,
+    DETAILED_WITH_PATHOLOGY,
+    LONGITUDINAL,
+    COMPLETE
+}
+
+enum class EfficacyEvidenceChapterType {
+    NONE,
+    STANDARD_OF_CARE_ONLY,
+    MOLECULAR_ONLY,
+    COMPLETE
+}
+
+enum class ClinicalChapterType {
+    NONE,
+    COMPLETE
+}
+
+enum class TrialMatchingChapterType {
+    NONE,
+    STANDARD_ALL_TRIALS,
+    STANDARD_EXTERNAL_TRIALS_ONLY,
```

Comment:
What does "standard" mean in this context?

Reply:
Standard means "default". Would "default" be a better term? I renamed `COMPLETE` to `DETAILED_ALL_TRIALS` to emphasise the difference with `STANDARD`